### ISSUE TITLE
Add C# wrappers to linux x86-64 binaries

### ISSUE THREAD

**Issue by DoubangoTelecom** [DoubangoTelecom/ultimateALPR-SDK#103]

**Comment by DoubangoTelecom** [comment]
For now only Windows x86-64 binaries are built with C# wrappers

**Comment by DoubangoTelecom** [comment]
If you want to add the C# interface by yourself:

```
gcc -c -fpic ultimateALPR-SDK-API-PUBLIC-SWIG_cscharp.cxx -IultimateALPR-SDK/c++ -LultimateALPR-SDK/binaries/linux/x86_64 -lultimate_alpr-sdk 
gcc -shared -o libultimateALPR-SDK.so ultimateALPR-SDK-API-PUBLIC-SWIG_cscharp.o
```

The output library name must be "libultimateALPR-SDK.so" to match "DllImport("ultimateALPR-SDK"," entries in https://github.com/DoubangoTelecom/ultimateALPR-SDK/blob/master/csharp/ultimateAlprSdkPINVOKE.cs.

**Comment by DoubangoTelecom** [comment]
Fixed in 3.0.0-rc6. The C# wrappers except a shared library named `libultimateALPR-SDK.so` which means you'll need to create a symlink on `libultimate_alpr-sdk.so` if you're using Linux.
